zerver/test_external.py

```diff
@@ -224,4 +264,3 @@ def test_change_user(self):
         self.login("hamlet@zulip.com")
         result = self.client.post('/json/users/me/android_gcm_reg_id', {'token':token})
         self.assert_json_success(result)
-
```

Comment:
Ahh, OK.  Probably possible to change the config, but it might make sense to change Zulip to not have trailing newlines to avoid this issue in the future.  Thanks for the info!  I opened https://github.com/zulip/zulip/issues/666 to track the newline issue.